edx_repo_tools/oep2/explode_repos_yaml.py

```diff
@@ -48,39 +49,93 @@ def explode(hub, dry):
 
         file_contents = yaml.safe_dump(repo_data, indent=4)
 
-        if dry:
-            click.secho(
-                'Against {}/{}'.format(user, repo_name),
-                fg='yellow', bold=True
-            )
-            click.secho(
-                'Would have created openedx.yaml file:',
-                fg='yellow', bold=True
-            )
-            click.secho(file_contents)
-            continue
+        file_contents = textwrap.dedent("""
+            # This file describes this Open edX repo, as described in OEP-2:
+            # http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+            {}
+        """).format(file_contents).strip()
 
         gh_repo = hub.repository(user, repo_name)
 
-        gh_repo.create_ref(
-            'refs/heads/{}'.format(BRANCH_NAME),
-            gh_repo.branch('master').commit.sha,
-        )
-        gh_repo.create_file(
-            path=OPEN_EDX_YAML,
-            message='Add an OEP-2 compliant openedx.yaml file',
-            content=file_contents,
-            branch=BRANCH_NAME,
-        )
-        pull = gh_repo.create_pull(
-            title='Add an OEP-2 compliant openedx.yaml file',
-            base='master',
-            head=BRANCH_NAME,
-        )
-        click.secho('Created pull request {} against {}'.format(
-            pull.html_url,
-            repo,
-        ), fg='green')
+        if gh_repo.fork:
+            LOGGER.info("Skipping %s because it is a fork", gh_repo.full_name)
+            continue
+
+        try:
+            parent_commit = gh_repo.default_branch.commit.sha
+        except:
+            LOGGER.warning(
+                "No commit on default branch %s in repo %s",
```

Comment:
Oh, sorry. That's a change I have locally (adding a `continue` to the except block).